crates/forge/tests/e2e/running.rs

```diff
@@ -958,37 +958,6 @@ fn should_panic() {
     );
 }
 
-#[test]
-fn printing_in_contracts() {
```

Comment:
Why was this removed?

Reply:
Copied from the PR description:
- Removed `printing_in_contracts` test since it doesn't test any foundry logic, but only Scarb output for a specific case

In more details, for some Scarb versions it returns a warning and for other it doesn't, which is a problem when we want to test different versions. Regardless, I think this test is simply not relevant to Foundry.